src/Process/Deploy/InstallUpdate/ConfigUpdate/SearchEngine/Config.php

```diff
@@ -83,7 +83,7 @@ public function get(): array
     /**
      * @return array
      */
-    private function getSearchConfig()
+    private function getSearchConfig(): array
```

Comment:
Add changes for ElasticSuite to make it visible in the configuration file.

Reply:
Please explain

Reply:
discussed offline